protected/humhub/assets/CoreBundleAsset.php

```diff
@@ -65,7 +66,8 @@ class CoreBundleAsset extends WebStaticAssetBundle
         LikeAsset::class,
         StreamAsset::class,
         ActivityAsset::class,
-        SpaceChooserAsset::class
+        SpaceChooserAsset::class,
+        SpaceSettingAsset::class,
```

Comment:
We should not add this to the CoreBundleAsset, the CoreBundleAsset should only contain bundles which are commonly and initially required at first page load (e.g. dashboard) or containing modules which are requried very often.